tests/autoclassify/test_classify_failures.py

```diff
@@ -50,7 +46,8 @@ def test_classify_test_failure(text_log_errors_failure_lines,
     for (error_line, failure_line), expected in zip(zip(*expected_classified),
                                                     classified_failures):
         assert list(error_line.classified_failures.values_list('id', flat=True)) == [expected.id]
-        assert list(failure_line.error.classified_failures.values_list('id', flat=True)) == [expected.id]
+        assert list(failure_line.error.classified_failures.values_list(
```

Comment:
Why do I see these changes?
Is your IDE formatting the files it touches?

Reply:
Yes I have an autoformatter, I'll disable it.